dev/TabView/TabView.xaml

```diff
@@ -364,33 +353,35 @@
     <Style TargetType="local:TabViewItem">
         <Setter Property="HorizontalContentAlignment" Value="Left"/>
         <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource TabViewItemCornerRadius}"/>
+        <Setter Property="BorderThickness" Value="{ThemeResource TabViewItemBorderThickness}"/>
+        <Setter Property="BorderBrush" Value="{ThemeResource TabViewItemBorderBrush}"/>
+        <Setter Property="Background" Value="{ThemeResource TabViewItemHeaderBackground}"/>
+        <Setter Property="MinHeight" Value="{ThemeResource TabViewItemMinHeight}"/>
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TabViewItem">
-
                     <Grid x:Name="LayoutRoot"
-                        Padding="{TemplateBinding Padding}"
-                        Margin="-1,0,0,0"> <!-- This item will cover the separator on its left side. -->
-
+                            Padding="{TemplateBinding Padding}">
                         <Grid.ColumnDefinitions>
-                            <ColumnDefinition x:Name="LeftColumn" Width="0"/>
+                            <ColumnDefinition x:Name="LeftColumn" Width="Auto"/>
                             <ColumnDefinition Width="*" />
-                            <ColumnDefinition x:Name="RightColumn" Width="0"/>
+                            <ColumnDefinition x:Name="RightColumn" Width="Auto"/>
```

Comment:
I think if you remove the arcs, you don't need these columns anymore, everything lives inside column 1 now.

Reply:
yes, you're right, will do